lib/fontcustom/generator/template.rb

```diff
@@ -201,6 +201,16 @@ def glyph_properties
   -moz-osx-font-smoothing: grayscale;
   -webkit-font-smoothing: antialiased;
   font-smoothing: antialiased;|
+        if @options[:ligature]
+          properties += %Q|
+  -webkit-text-rendering: optimizeLegibility;
+  -moz-text-rendering: optimizeLegibility;
+  -ms-text-rendering: optimizeLegibility;
+  -o-text-rendering: optimizeLegibility;
+  text-rendering: optimizeLegibility;|
```

Comment:
I know a performance problem of `optimizeLegibility`. but, ligature fonts needs this property for rendering glyphs on browser. if you set ligature options to false, remove this properties.